pkgs/by-name/wr/wrkflw/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  docker,
+}:
+
+rustPlatform.buildRustPackage rec {
```

this rec in unnecessary 
```suggestion
rustPlatform.buildRustPackage {
```